layouts/protected/brand-protected-screen.tsx

```
@@ -0,0 +1,94 @@
+    // If the manager has no brands, redirect to onboarding
+    router.replace({
+      pathname: "/onboarding-your-brand",
+      params: {
+        firstBrand: "true",
+      },
+    });
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    if (manager && manager.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)